packages/front-end/pages/metric/[mid].tsx

```diff
@@ -14,9 +14,6 @@ import {
   formatConversionRate,
   defaultWinRiskThreshold,
   defaultLoseRiskThreshold,
-  defaultMaxPercentChange,
-  defaultMinPercentChange,
-  defaultMinSampleSize,
```

Comment:
Should we just add those two other metric defaults into this PR as well?  Seems weird to leave them as constants here when we have a `MetricDefaults` type and hook now.